test/transform/output.js

```diff
@@ -0,0 +1,42 @@
+var _templateReference = {},
+  _templateReference2 = {},
+  _templateReference3 = {},
+  _templateReference4 = {};
+const div = ESXToken.template(
```

Comment:
maybe not relevant at this stage but `ESXToken` should be defined *once* globally if not defined yet and per program, a bit like Babel core utilities, where these are defined once (`_extends` or similar ... `ESXToken` is like a polyfill so that `globalThis.ESXToken || (globalThis.ESXToken = ...);` should be present somewhere or this transformer won't create standalone results ... or maybe I am missing something

Reply:
We could either inject it inline (like Babel does for most helpers), or we could inject an `import` statement to an `@ungap/estoken` polyfill.

The first one works everywhere, but the second one avoids code duplication in bundles.

Reply:
Maybe an option `polyfill: false | "inline" | "import"`.

Reply:
if the *inline* duplicates the import all over the place (e.g. per each file) I'm more than keen to consider the *import* version for code-size sake ... but I definitively like the `polyfill` option too ... anything that doesn't take too much effort works at this stage, but if it's not too complicated to get the polyfill I'll create already that ungap/esx repo and publish it

Reply:
I'm working on the option :+1:

Reply:
I've just published [@ungap/esxtoken](https://github.com/ungap/esxtoken) based on [this implementation](https://gist.github.com/WebReflection/2d64f34cf58daa812ec876242c91a97c?permalink_comment_id=4363545#gistcomment-4363545)